spec/index.bs

```diff
@@ -2530,6 +2522,66 @@ enum GPUBindingType {
 };
 </script>
 
+Each {{GPUBindingType}} has an associated {{GPUBindingResource}} type and [=internal usage=], given
+by this table:
+
+<table class="data">
+  <thead>
+    <tr>
+      <th>{{GPUBindingType}}</th>
+      <th><dfn dfn>Binding resource type</dfn></th>
+      <th><dfn dfn>Binding usage</dfn></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>{{GPUBindingType/"uniform-buffer"}}</td>
+      <td>{{GPUBufferBinding}}</td>
+      <td>[=internal usage/constant=]</td>
+    </tr>
+    <tr>
+      <td>{{GPUBindingType/"storage-buffer"}}</td>
+      <td>{{GPUBufferBinding}}</td>
+      <td>[=internal usage/storage=]</td>
+    </tr>
+    <tr>
+      <td>{{GPUBindingType/"readonly-storage-buffer"}}</td>
+      <td>{{GPUBufferBinding}}</td>
+      <td>[=internal usage/storage-read=]</td>
+    </tr>
+    <tr>
+      <td>{{GPUBindingType/"sampler"}}</td>
+      <td>{{GPUSampler}}</td>
+      <td></td>
```

Comment:
I don't think there's a strict need for it, but we could call these "constant" I think.

Reply:
Done